client/src/components/Navigation/MobileNavBar/MobileNavBar.js

```diff
@@ -64,7 +64,7 @@ const MobileNavBar = ({ menu }) => {
       <Dropdown overlay={profileMenu} trigger={['click']}>
         <span className='profilepic'>
           {currentUser && (
-            <img src={currentUser.avatarURL} className='profilepic' alt='profile pic' />
+            <Avatar src={currentUser.avatarURL} />
```

Comment:
Was there a corresponding css class that could’ve been deleted too?

Reply:
nvm no it needs to stay because it's also used in the span